src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/mainscreen/view/MainScreenView.kt

```diff
@@ -112,12 +112,14 @@ class MainScreenView : View() {
         //navigate back to verse selection from viewing takes
         if (viewModel.selectedContentProperty.value != null) {
             viewModel.selectedContentProperty.value = null
+            activeFragment.navigateBack()
         }
         //from verse selection, navigate back to chapter selection
         else if (viewModel.selectedCollectionProperty.value != null) {
```

Comment:
Why shouldn't navigateBack() be called in this case?

Reply:
because when we navigate from the chapter cards to the verse/chunk cards we don't actually dock a new fragment. Collection grid and content grid were consolidated into one view model, and then we have one `card data` data type which was determining the data that is displayed in the datagrid. so when a user navigated from the chapters to verses the underlying data was changing but the fragment stayed the same